lib/src/dom/anchors.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Generated from Web IDL definitions.
+
+@JS()
+library;
+
+import 'dart:js_interop';
+
+import 'webxr.dart';
+
+extension type XRAnchor._(JSObject _) implements JSObject {
```

Comment:
https://caniuse.com/mdn-api_xranchor

Still considered experimental 🤷

Reply:
Yup, another experiment I did locally was `standardsTrack && !experimental`. That was a net of +2000 or so and includes Trusted Types.